decred/decred/dcr/rpc.py

```diff
@@ -9,9 +9,11 @@
 
 from decred.util import tinyhttp
 from decred.util.encode import ByteArray
+from decred.util.helpers import satsToCoins as coinify
```

Comment:
This seems like a clue that you should just name the function `coinify` to begin with.